functions/adjustChatCapacity.ts

```diff
@@ -32,7 +32,8 @@ type EnvVars = {
 
 export type Body = {
   workerSid?: string;
-  adjustment?: 'increase' | 'decrease' | 'increaseUntilCapacityAvailable';
+  // 'increase' and 'decease' can be removed once backend manual pulling is enabled everywhere
+  adjustment?: 'increase' | 'decrease' | 'increaseUntilCapacityAvailable' | 'setTo1';
```

Comment:
Do we still use the adjustment `decrease` in any scenarios?

Reply:
Just Flex running with `enable_backend_manual_pulling` set to false. It can be removed once everything is using backend pulls